src/Chapter2.hs

```diff
@@ -841,7 +879,13 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [Int] -> [Int]
+rotate _ [] = []
+rotate _ [x] = [x]
```

Comment:
It looks like this case doesn't return an empty list for negative numbers :thinking: